src/prob/opf.jl

```diff
@@ -118,7 +118,7 @@ function build_mc_opf(pm::AbstractUBFModels)
     # Variables
     variable_mc_bus_voltage(pm)
     variable_mc_branch_current(pm)
-    variable_mc_branch_power(pm)
+    variable_mc_branch_power(pm; bounded=false)
```

Comment:
can you add a `# TODO: revert to bounded in v0.10` at the end of this line here?